specification/metadata/datageneratorversion.md

```diff
@@ -0,0 +1,17 @@
+# Data Generator Version
+
+A version specific to the data generator for providers which support multiple FOCUS datasets with different provider columns, or for providers who release multiple versions of their generated datasets between FOCUS Versions.
```

Comment:
I understand that a data generator would have different options to include or not different columns in the dataset, and that the generator version would include (reflect) the usage of these options. Is that the intention?

If yes, wouldn't it be better to have separate columns for the version (released software version) of the data generator and for its usage (options used when calling the generator)?

Reply:
Im not following your suggestion. Can you please be more concrete with a suggested edit

Reply:
Maybe simply:

> A version number that identifies a unique state (or release?) of the Data Generator. Versioning enables practitioners to identify which FOCUS version is supported by which Data Generator Version. Multiple Data Generator Versions may support the same FOCUS version, corresponding to different releases to fix bugs or to introduce additional provider columns.

Maybe also:
> The Data Generator Version SHOULD use semantic versioning. 

NOTE: I would tend to think the Data Generator Version could be removed in favor of the Schema ID: https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/390#discussion_r1570240393